buddyboss/buddyboss-platform
# Option to hide all sub-groups from the main Groups directory

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1079 .

### How to test the changes in this Pull Request:

1. BuddyBoss > Settings > Groups, in the "Group Hierarchies" metabox.
2. When you enable "Group Hierarchies" see another option "Hide Subgroups"
3. Enable and check the group directory.

### Proof Screenshots or Video
http://prntscr.com/td7kwy
Enable: http://prntscr.com/td7lby
Disable: http://prntscr.com/td7lr3


### Other information:
